2003-07-11  Elena Zannoni  <ezannoni@redhat.com>

        * lib/gdb.exp (setup_kfail, kfail): Redefine procedures.
                                                                               
--- ./gdb/testsuite/lib/gdb.exp.1	2004-11-24 15:59:46.131394720 -0500
+++ ./gdb/testsuite/lib/gdb.exp	2004-11-24 16:01:06.304206600 -0500
@@ -63,6 +63,15 @@
 
 ### Only procedures should come after this point.
 
+if {![llength [info procs kfail]]} {
+    proc setup_kfail { args } {
+	#setup_xfail args
+    }
+    proc kfail { bugid message } {
+	fail $message
+    }
+}
+
 #
 # gdb_version -- extract and print the version number of GDB
 #
